### PR TITLE
Use systemctl enable --now

### PR DESCRIPTION
**Pull request by ekohl**
Using systemctl enable --now is equivalent to enable and start. It is already used in some places in the documentation, but this makes it consistent. The benefit for users is that they have to copy fewer commands.

It's still a draft since there are some oddities which I'll highlight.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_configuring-a-host-for-registration.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ You must update each virtual machine configuration so that they receive updates
 ** For {EL} 7:
 +
 ----
-# systemctl start rhsmcertd
+# systemctl enable --now rhsmcertd
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-a-host-for-registration.adoc:32]: Previously this was only start, but not enable while for EL 6 it did both enable and start. I think this is fixing a bug.

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc [hunk 2]
```diff
@@ -122,15 +122,15 @@ _990_
...
+# systemctl start nfs-lock nfs-idmapd
 ----
 endif::[]
 ifdef::foreman-deb[]
 ----
 # {package-install} nfs-kernel-server nfs-common
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-idmapd
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-an-external-dhcp-server.adoc:133]: It's weird that some services are started but not enabled, which means that it's broken after a reboot? Not sure what would be best.

At least on Fedora it looks like `nfs-server` only has `Wants` on `nfs-idmapd` but no `Requires`. Possible bug?

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc [hunk 3]
```diff
@@ -70,7 +70,7 @@ euid = __ID_of_Apache_User__
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # systemctl restart gssproxy.service
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:72]: The restart here is weird. Is there a specific reason for this?

Comment (evgeni) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:72]: It was added in the "original" version of that doc (in https://bugzilla.redhat.com/show_bug.cgi?id=1206138) but I don't see any real reason for that, no.

However, looking at a system that I have here, which *does not have IPA enabled at all*, the `gssproxy` service is running, even if it's marked as not enabled:

```console
# systemctl status gssproxy
● gssproxy.service - GSSAPI Proxy Daemon
   Loaded: loaded (/usr/lib/systemd/system/gssproxy.service; disabled; vendor preset: disabled)
   Active: active (running) since Fri 2022-11-25 07:07:56 UTC; 1h 47min ago
 Main PID: 745 (gssproxy)
    Tasks: 6 (limit: 50696)
   Memory: 1.6M
   CGroup: /system.slice/gssproxy.service
           └─745 /usr/sbin/gssproxy -D
```

Some steps above we do edit the config, so if the service is already running, a restart is really required.

Comment (ekohl) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:72]: I'm guessing something activates it. Either via a socket or something else via a requirement. Let's be safe and keep restart in then. It doesn't hurt in this order (first `enable --now` and then `restart` could cause a double service start, which is slower).

Comment (evgeni) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:72]: I could not find *what* activated it, but yeah there must be something.

And agreed, it doesn't hurt that way, so let's keep it

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc (hunk 2 above)
Comment (evgeni) [reply guides/common/modules/proc_configuring-an-external-dhcp-server.adoc:133]: This is Debian, which enables services by default. But I do agree, just put it to the `enable --now` line.

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc [hunk 4]
```diff
@@ -122,15 +122,15 @@ _990_
 ifndef::foreman-deb[]
 ----
 # {package-install} nfs-utils
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-lock nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-lock nfs-idmapd
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-an-external-dhcp-server.adoc:126]: what's `nfs-lock`, neither my EL8 nor my F36 systems know such a service?!

ah, looking at an EL7 system:
```console
# systemctl status nfs-lock
● rpc-statd.service - NFS status monitor for NFSv2/3 locking.
   Loaded: loaded (/usr/lib/systemd/system/rpc-statd.service; static; vendor preset: disabled)
   Active: inactive (dead)
# ls -alh /usr/lib/systemd/system/nfs-lock.service 
lrwxrwxrwx. 1 root root 17 Dec 21  2020 /usr/lib/systemd/system/nfs-lock.service -> rpc-statd.service
```

So I guess this should be swapped out, and also be moved to `enable --now`?

Can we then just fold the two code sections into one, only differentiating on the package line?

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc [hunk 5]
```diff
@@ -122,15 +122,15 @@ _990_
...
 ----
 endif::[]
 ifdef::foreman-deb[]
 ----
 # {package-install} nfs-kernel-server nfs-common
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-an-external-dhcp-server.adoc:131]: `nfs-kernel-server` depends on `nfs-common`, so I see no good reason to explicitly call that out? (scope creep, sorry)

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc (hunk 4 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-an-external-dhcp-server.adoc:126]: It looks like `nfs-server` has wants on `rpcbind` and `nfs-idmapd` so they will be started. Should we reduce the list of services to just `nfs-server`? If you only want an NFSv4 server that's really all you need according to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/configuring-an-nfsv4-only-server_managing-file-systems

File: guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc [hunk 6]
```diff
@@ -8,19 +8,12 @@ You can configure {ProductName} with an external DHCP server.
 For more information, see xref:configuring-an-external-dhcp-server_{context}[].
 
 .Procedure
-. Install the `nfs-utils` utility:
+. Install the `{nfs-client-package}` utility:
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc:11]:
```suggestion
. Install the `{nfs-client-package}` package:
```

It's hardly *a* utility, more a collection thereof.

**Review by evgeni — APPROVED** [review]
One wording nitpick, but not blocking, ack

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   fb6a16ce5..9ef18fd4d  3.3 -> 3.3
   8631e690f..7dbde1198  3.4 -> 3.4
   6e68f4be6..49ed39e61  3.5 -> 3.5
